### PR TITLE
Docs: Add Vignette and update readme

### LINKED ISSUE #22: Add Vignette and publish package to website

### PR DESCRIPTION
**Pull request by mel-liow**
- [X] Adds vignette markdown file (see screen recording of output)
- [X] Adds more documentation to Usage in the README.md
- [X] Add build for hosting package to github pages

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -74,9 +74,36 @@ devtools::install_github("UBC-MDS/rpollution")
...
+
+**Retrieve historic pollution data with specified date range and location:**
+``` r
+get_pollution_history(1606488670, 1606747870, 49.28, 123.12, api_key)
+```
+
+**Generate an interactive map containing current pollution data by location:**
+
+``` r
+get_air_pollution(49.28, 123.12, api_key, "Current Air Pollution")
```
Comment (danfke) [inline README.md:92]: There is an optional fig_title argument which might be worth showing off in the vignette

File: vignettes/rpollution.Rmd [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+```
+
+```{r historic-example-result}
+head(df)
+```
+
+## Generate an interactive map containing current pollution data by location:
+
+```{r current-example}
+pollution_map_plot <- get_air_pollution(49.28, 123.12, api_key)
```
Comment (danfke) [inline vignettes/rpollution.Rmd]: There is an optional fig_title argument which might be worth showing off in the vignette

File: DESCRIPTION [hunk 3]
```diff
@@ -27,3 +27,7 @@ Imports:
     jsonlite,
     magrittr,
     testthat
+Suggests:
```
Comment (danfke) [inline DESCRIPTION:30]: Have you run check() to make sure the package still work?

**Review by danfke — APPROVED** [review]

File: DESCRIPTION (hunk 3 above)
Comment (mel-liow) [reply DESCRIPTION:30]: yup! Check still works
Screenshot 2022-01-31 at 17 54 20

File: vignettes/rpollution.Rmd (hunk 2 above)
Comment (mel-liow) [reply vignettes/rpollution.Rmd]: Done!

Comment (danfke) [reply vignettes/rpollution.Rmd]: Oh sorry I was referring to the fig_title argument in my function lol

**Comment by danfke** [comment]
Also I think the vignette is supposed to have more of a narrative, like in the dplyr https://cran.r-project.org/web/packages/dplyr/vignettes/dplyr.html linked to in lecture... Sorry can't remember if that is what Chris is doing

File: vignettes/rpollution.Rmd (hunk 2 above)
Comment (mel-liow) [reply vignettes/rpollution.Rmd]: Oh gotcha, okay will change it

**Comment by mel-liow** [comment]
@danfke I've added more to the vignette if you'd like to take another look! 

https://user-images.githubusercontent.com/40150248/152016471-b6df93ee-1b5d-4eb4-a172-2493a4324e49.mov

**Review by danfke — APPROVED** [review]
Thanks for making the changes Mel it looks great!

**Pull request merged**